content/en/docs/client-options.md

```diff
@@ -62,6 +62,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [ACME-PS](https://github.com/PKISharp/ACMESharpCore-PowerShell)
 - [FreeSSL.tech Auto](https://freessl.tech)
 - [acme-distributed](https://github.com/jannfis/acme-distributed)
+- [LECert](https://github.com/skoerfgen/LECert)
```

Comment:
Hi @skoerfgen,

Thanks for writing a client and a PR.

The "LE" in "LECert" is going to be read as Let's Encrypt and I've been told it doesn't meet our trademark policy: https://letsencrypt.org/trademarks/

Would you be willing to rename your client to "ACMECert" or equivalent? It would meet our trademark requirements and also be a more accurate name since your code will work with other compatible ACME CAs :-)

Thanks!

Reply:
Hello @cpu,

Sure no problem. I took the name you suggested :)

Thanks!